index.js

```diff
@@ -43,17 +38,32 @@ module.exports = (robot, options, visit) => {
     });
   }
 
+  // Trigger the event
+  function trigger(event) {
+    robot.webhook.emit(event.event, event);
+  }
+
   function schedule(installation, repository) {
```

Comment:
That's a good point. I haven't really had a need to customize the interval yet (currently checks hourly, but could probably be daily for my use case), so I'm honestly not sure the best API for this.

I'm tempted to wait to change it whenever custom intervals are introduced, but that will be another breaking change.

What do you think?